docs/windows.md

```diff
@@ -1,5 +1,29 @@
 # Deploying Antrea on Windows
 
+## Table of Contents
+
+<!-- toc -->
+- [Overview](#overview)
+  - [Components that run on Windows](#components-that-run-on-windows)
+  - [Antrea Windows demo](#antrea-windows-demo)
+- [Deploying Antrea on Windows Worker Node](#deploying-antrea-on-windows-worker-node)
+  - [Prerequisites](#prerequisites)
+  - [Installation as a Service (Containerd based runtimes)](#installation-as-a-service-containerd-based-runtimes)
+  - [Installation as a Pod (Docker/Containerd based runtimes)](#installation-as-a-pod-docker/containerd-based-runtimes)
+    - [Download & Configure Antrea for Linux](#download-&-configure-antrea-for-linux)
+    - [Add Windows antrea-agent DaemonSet](#add-windows-antrea-agent-daemonset)
+    - [Join Windows worker Nodes](#join-windows-worker-nodes)
+    - [1. (Optional) Install OVS (provided by Antrea or your own)](#1.-optional-install-ovs-provided-by-antrea-or-your-own)
+    - [2. Disable Windows Firewall](#2.-disable-windows-firewall)
+    - [3. Install wins, kubelet, kubeadm and configure kubelet startup params](#3.-install-wins-kubelet-kubeadm-and-configure-kubelet-startup-params)
+    - [4. Prepare Node environment needed by antrea-agent](#4.-prepare-node-environment-needed-by-antrea-agent)
+    - [5. Run kubeadm to join the Node](#5.-run-kubeadm-to-join-the-node)
+    - [Verify your installation](#verify-your-installation)
+  - [Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)](#add-windows-kube-proxy-daemonSet-only-for-kubernetes-versions-prior-to-1.26)
```

Comment:
I don't know if it's a bug with a ToC generator or an issue on my side, but this link doesn't seem to be working.